tests/test_main.py

```diff
@@ -7,7 +7,7 @@
 
 @pytest.fixture(autouse=True)
 def setup_auth_mock(auth_mock):
```

Comment:
Do we still need this?

Reply:
This was needed for overcoming the pylint unused argument warning, I added as an ignore line.